lib/react-rails.rb

```diff
@@ -2,3 +2,5 @@
 require 'react/jsx'
 require 'react/rails'
 require 'react/server_rendering'
+
+ActionDispatch::IntegrationTest.send(:include, React::Rails::TestHelper)
```

Comment:
I think there's a better way of including modules in rails without using the `send` keyword, though I've not used it much myself; Devise does it in this way:
https://github.com/plataformatec/devise/blob/master/lib/devise/rails.rb#L21
and
https://github.com/plataformatec/devise/blob/master/lib/devise.rb#L12

Reply:
I just updated for require test_helper by autoload, include helper by use `ActiveSupport.on_load`